source/reference/known-issues.txt

```diff
@@ -68,3 +68,24 @@ Given the exact same |sql| query, the |bi-short| does not return
       GROUP BY class;
       
       Empty set (0.00 sec)
+
+
+BI Connector Returns ``NULL`` Values due to Mapping Error
+----------------------------------------------------------
+
+A known mapping error can occur that causes all returned values to be ``NULL`` 
+in the result. This error occurs when you query a collection that
+contains an array field that can be ``NULL``, and the same array contains 
+an additional field that can also be ``NULL``. A workaround for this error is to 
+create a view by including the ``nonnullsched`` parameter with the ``createView`` method.
```

Comment:
Can you add some context on what this view returns and why this is a valid workaround? From the ticket description:

> The workaround is to create a view that contains only documents where the array field is an array: